CMakeLists.txt

```diff
@@ -610,7 +611,18 @@ endif()
 
 if(USE_OPENCV)
   add_executable(im2rec "tools/im2rec.cc")
-  target_link_libraries(im2rec ${BEGIN_WHOLE_ARCHIVE} mxnet ${END_WHOLE_ARCHIVE} ${mxnet_LINKER_LIBS} ${OpenCV_LIBS} dmlc)
+  if(MSVC)
+    target_link_libraries(im2rec mxnet)
+  else()
+    target_link_libraries(im2rec ${BEGIN_WHOLE_ARCHIVE} mxnet_static ${END_WHOLE_ARCHIVE})
+  endif()
+  target_link_libraries(im2rec
+    ${mxnet_LINKER_LIBS}
+    ${OpenCV_LIBS}
+    dmlc
+    ${nnvm_LINKER_LIBS}
+    ${pslite_LINKER_LIBS}
+    )
```

Comment:
Can you explain this change? How is related to the original patch?

Reply:
@cjolivier01 could you explain why you added this?